.config/python_version.config

```diff
@@ -0,0 +1 @@
+PYTHON_VERSION=3.10
```

Comment:
We're currently in 3.11.4 :)

Reply:
done here 
https://github.com/oxsecurity/megalinter/pull/2737/commits/e112cb0146277d59410e3261572e087bfbd09251